pages/1.10/installing/production/deploying-dcos/installation/index.md

```diff
@@ -114,38 +114,38 @@ In this step, an IP detection script is created. This script reports the IP addr
 
         [enterprise type="inline" size="small" /]
 
-```bash
-#!/usr/bin/env bash
-set -o nounset -o errexit
-MASTER_IP=172.28.128.3
-echo $(/usr/sbin/ip route show to match 172.28.128.3 | grep -Eo '[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}' | tail -1)
-```
+        ```bash
```

Comment:
Were the leading spaces for lines 117-148 intended to be added? It looks fine when I view the file but I just want to make sure.

Reply:
I adjusted it to display properly in line with the instructions, which are also indented